src/garage/np/policies/base.py

```diff
@@ -1,68 +1,88 @@
-from garage.core import Parameterized
+"""Base class for policies based on numpy."""
+import abc
 
 
-class Policy(Parameterized):
+class Policy(abc.ABC):
+    """Base classe for policies based on numpy."""
+
     def __init__(self, env_spec):
-        Parameterized.__init__(self)
         self._env_spec = env_spec
 
     # Should be implemented by all policies
-
+    @abc.abstractmethod
     def get_action(self, observation):
+        """
+        Get action sampled from the policy.
+
+        Args:
+            observation (np.ndarray): Observation from the environment.
+        Returns:
+            action (np.ndarray): Action sampled from the policy.
+
+        """
         raise NotImplementedError
 
     def reset(self):
+        """Reset the policy."""
         pass
 
     @property
     def observation_space(self):
+        """Return the observation space of the environment."""
```

Comment:
properties don't need the verb "Return"

Reply:
properties also need types

Reply:
see https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html for examples